kde-kit/merge.kit.d/base_autogen.yml

```diff
@@ -0,0 +1,31 @@
+sources:
+# We need this to inherit the eclass used for kit cache JSON generation
+- name: "core-kit"
+  url: "https://github.com/macaroni-os/core-kit"
+  branch: "mark-unstable"
+
+target:
+  name: kde-kit
+  url: "https://github.com/macaroni-os/kde-kit"
+  branch: mark-unstable
+
+  fixups:
+    include:
+      - file: ../../LICENSE
+        to: LICENSE
+
+      - file: ../../.github/FUNDING.yml
+        to: .github/FUNDING.yml
+
+  atoms_defaults:
+    max_versions: 3
+
+  atoms:
+    - pkg: app-crypt/qca
+    - pkg: dev-libs/plasma-wayland-protocols
+    - pkg: dev-libs/qcoro
+    - pkg: dev-libs/xerces-c
```

Comment:
Could be better add xerces-c in `dev-kit`? it seems a library not strictly used only for KDE.